app/tasks/education_form/create_daily_spool_files.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+module EducationForm
+  class CreateDailySpoolFiles
```

Comment:
same, if it's on a daily or semi daily schedule then maybe something with whenever (which taps into cron), to execute rails runner tasks is how I have done this sort of thing in the past.

However, if you need fault tolerance / retry capability then something like sidekiq + redis makes sense and I would just put it in a "less critical" queue and assign it certain time to run, 10.hours.from_now or something.